sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-response/_tests_/ece-response-integration.scheduler.e2e-spec.ts

```diff
@@ -359,6 +532,7 @@ describe(
       expectedResult.summary = [
         `Starting download of file ${confirmEnrolmentResponseFile}.`,
         `The file ${confirmEnrolmentResponseFile} has been deleted after processing.`,
+        "Notification has been created to send email to integration contacts.",
```

Comment:
I am not seeing the notification check here, is that the intention here? then why description was changed? is it for this line

Reply:
Notification check is missed. will add it.